### ISSUE TITLE
NTR: Costovertebral angle tenderness (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7962]
New term request Costovertebral angle tenderness (cohe11_181018172855): CVA tenderness; flank tenderness; Renal angle  UMLS: 0235634

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000106
- label: Costovertebral angle tenderness
- parents: [Pain (HP:0012531)](https://hpo.jax.org/app/browse/term/HP:0012531)
- pmid: PMID:36605912
- def: Pain that is elicited by palpation of the region inside of the costovertebral angle, which is formed by the 12th rib and the spine.
- comment: Costovertebral angle tenderness can be observed with conditions including pyelonephritis and ureteral stones.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7962
- synonyms: Costovertebral angle pain(NONE)
